src/main/mdo/dependency-updates-report.mdo

```diff
@@ -0,0 +1,177 @@
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<model xmlns="http://codehaus-plexus.github.io/MODELLO/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="http://codehaus-plexus.github.io/MODELLO/2.0.0 https://codehaus-plexus.github.io/modello/xsd/modello-2.0.0.xsd"
+       xsd.namespace="http://mojo.codehaus.org/versions-maven-plugin/rule/2.0.0"
+       xsd.targetNamespace="http://mojo.codehaus.org/versions-maven-plugin/rule/2.0.0">
+    <id>dependency-updates-report</id>
+    <name>DependencyUpdatesReport</name>
+    <description>
+        XML version of the Dependency Updates Report
+    </description>
+
+    <defaults>
+        <default>
+            <key>package</key>
+            <value>org.codehaus.mojo.versions.reporting.model</value>
+        </default>
+    </defaults>
+
+    <versionDefinition>
+        <type>namespace</type>
+    </versionDefinition>
+
+    <classes>
+
+        <class rootElement="true" xml.tagName="DependencyUpdatesReport">
+            <name>DependencyUpdatesReport</name>
+            <fields>
+                <field>
+                    <name>summary</name>
+                    <required>true</required>
+                    <association>
+                        <type>DependencyReportSummary</type>
+                        <multiplicity>1</multiplicity>
+                    </association>
+                </field>
+                <field>
+                    <name>dependencyManagements</name>
+                    <association xml.itemsStyle="wrapped" xml.tagName="dependencyManagement">
+                        <type>DependencyInfo</type>
+                        <multiplicity>*</multiplicity>
+                    </association>
+                </field>
+                <field>
+                    <name>dependencies</name>
+                    <association xml.itemsStyle="wrapped" xml.tagName="dependency">
+                        <type>DependencyInfo</type>
+                        <multiplicity>*</multiplicity>
+                    </association>
+                </field>
+            </fields>
+        </class>
+
+        <class>
+            <name>DependencyReportSummary</name>
+            <fields>
+                <field>
+                    <name>usingLastVersion</name>
+                    <required>true</required>
+                    <type>String</type>
```

Comment:
So, I'd really like to have an `int` there, but all serializer implementations will refrain from producing an element there if the value is equal to zero; they seem to treat zero as a non-value. I guess that might call for a change request in Modello?

Reply:
Or maybe replace Modello with something else: e.g. Jackson or JAXB?

Reply:
I would like to stay with modelo, not introduce too much idea in one time